### ISSUE TITLE
[Proposal] Do not autoreload page after any action in the issue management

### ISSUE THREAD

**Issue by delvh** [go-gitea/gitea#15509]
- Gitea version (or commit ref): 1.15.0+dev-152-gc29e85228 and below (1.14.1)
- Can you reproduce the bug at https://try.gitea.io:
  - Yes [](https://try.gitea.io/delvh/kanban-test/issues/14)

## Description
Currently, after almost any action for an issue (simply closing the label dropdown, adding/ clearing a project from/ to an issue, … ), the page gets reloaded, which proves quite time intensive when trying to manage issues and having to wait 1-2 seconds between each action.
I think it would be better if the changes would be posted silently without always reloading the page.
I also think that is a bug because I do not remember this kind of behavior before v1.14.0/1.14.1 (don't know which exactly).

**Comment by lunny** [comment]
We should switch reload the page to ajax.
